scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala

```diff
@@ -78,7 +75,7 @@ trait HasTests extends FormatAssertions {
     val sep =
       if (content.contains(System.lineSeparator)) System.lineSeparator
       else "\n"
-    val spec = filename.stripPrefix(testDir + File.separator)
+    val spec = filename.stripPrefix(BuildInfo.resourceDirectory.toURI.toString)
```

Comment:
it will only work if `toURI` ends with `/` (or `\` on Windows), right? to have a relative name.

Reply:
The logic here was buggy. I refactored it to use only `java.nio.file.Path`